index.html

```diff
@@ -192,6 +192,34 @@ <h1>Examples</h1>
 pointerEventInitDict.coalescedEvents = [p1, p2];
 const event2 = new PointerEvent("pointermove", pointerEventInitDict);
 eventTarget.dispatchEvent(event2);</code>
+</pre>
+<pre id="example_5" class="example" title="Assigning a pen color on PointerDown">
+    <code>&lt;div style="position:absolute; top:0px; left:0px; width:100px;height:100px;"&gt;&lt;/div&gt;
+    &lt;script&gt;
+    window.addEventListener("pointerdown", assignPenColor);
+    window.addEventListener("pointerMove", assignPenColor);
```

Comment:
nit: pointermove should be all lowercase.

Reply:
Done